### PR TITLE
Update Enable-SPSessionStateService.md

### PR DESCRIPTION
**Pull request by rainerasbach**
Remove the spaces from the database name in the example 

Additional comment why not to use spaces in the database name

### CODE REVIEW

**Review by get-itips — APPROVED** [review]
While I am ok with the change, I would avoid using "file name" and just use "name"

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward @dariomws

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

File: sharepoint/sharepoint-server-ps/sharepoint-server/Enable-SPSessionStateService.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ This example enables a ASP.NET session state on a SharePoint Server farm that us
 Specifies the name of the database for the session state service.
 
 The type must be a valid name of a SQL Server database; for example, SessionStateDB1.
+Avoid using spaces in the DatabaseName. Using a file name like 'Session State Database' will cause problems adding new servers to the SharePoint farm.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-server-ps/sharepoint-server/Enable-SPSessionStateService.md:61]:
```suggestion

Avoid using spaces in the database name. Using a file name like 'Session State Database' will cause problems adding new servers to the SharePoint farm.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@rainerasbach, please commit the change as per @tseward's suggestion and add an extra line space. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@rainerasbach

**Comment by rainerasbach** [comment]
Hi Trevor,

I just had a support case that was caused in the first place by a “Session State Database” (with blanks in the name)
The customer could not add an additional server to the farm as long as he had this database name for his Session State service.
When the customer disabled the service he could add a new server.
When the customer enable the service again with a name without space, he could add a server.

I was able to fully reproduce this behavior.

This behavior is documented as well in Scaling out an installation, configuration on new app server fails: Task configdb has failed with an unknown exception (microsoft.com)<https://social.technet.microsoft.com/Forums/en-US/7e905658-3c2a-49ce-ade0-dcc439c1eab1/scaling-out-an-installation-configuration-on-new-app-server-fails-task-configdb-has-failed-with-an?forum=sharepointgeneralprevious>

I think the sentence would help explaining the situation and giving users a good reason not to put spaces in the database name, but if you don’t want this sentence I the article, remove it, but publish the change in the sample.

Kind regards
  Rainer

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tseward Gentle reminder.

**Comment by rainerasbach** [comment]
Done. The extra sentence was removed.

**Pull request merged**